src/js/hyperschedule.ts

```diff
@@ -527,9 +527,51 @@ function coursePassesTextFilters(
   return true;
 }
 
-function parseDaysInequality(inputDays: string) {
-  for (const rel of filterInequalities)
-    if (inputDays.startsWith(rel)) return rel;
+// timeFilters is a one or two element array
+// [start_time] or [start_time, end_time]
+function coursePassesTimeFilters(
+  course: Course.CourseV3,
+  timeFilters: string[]
+) {
+  // indicates no current time filters
+  if (timeFilters[0] === "") {
+    return true;
+  }
+
+  const startTime = parseFloat(timeFilters[0].replace(":", "."));
+  // if multiple start times, specified start must match at least one
+  if (timeFilters.length === 1) {
+    for (const schedule of course.courseSchedule) {
+      let scheduleStart = schedule.scheduleStartTime.replace(":", ".");
+      if (startTime === parseFloat(scheduleStart)) {
```

Comment:
Do these allow for inequality queries (e.g. `>9:35`)?  It doesn't seem apparent to me that it does.

Reply:
Either this should be fixed, or the documentation in the "help" popup should be corrected.

Reply:
The inequalities are actually handled in `getTimeRange`. Depending on the inequality, the time filter pushes an upper or lower bound of "00:00" or "23:59".